docs/automated-process.md

```diff
@@ -71,13 +71,18 @@ Note that most of the following items cannot be done by yourself, but need someo
     The notification includes a link to the page where the upload step can be
     approved.
 
-2. Check out the PR in your `ansible-build-data` clone and copy the updated
-   porting guide from its `${MAJOR_VERSION}` directory into the
-   [`docs/docsite/rst/porting_guides/`](https://github.com/ansible/ansible-documentation/tree/devel/docs/docsite/rst/porting_guides/)
-   directory of the [`ansible-documentation`
-   repository](https://github.com/ansible/ansible-documentation/). Create a
-   PR for updating the porting guide for the `devel` branch of
-   `ansible-documentation`.
+2. To create the porting guide PR in the ansible-documentation repository, trigger [the automated
+   workflow](https://github.com/ansible/ansible-documentation/actions/workflows/release-porting-guide.yml)
+   on the **Actions** tab of the repository. This workflow has the following inputs,
+
+    * **Release Branch name**. Specify the name of the branch from the newly created [ansible-build-data](https://github.com/ansible-community/ansible-build-data/) PR for the release, for example: `refs/pull/576/merge`.
+    * **Exact release version**. Specify a release version such as `11.2.0` or `12.0.0rc1`.
+    * **Use the workflow from** the **Branch: devel** `devel` is selected as the branch by default. Do not edit this while doing the release. This option is there to test the workflow itself.
+
+   The process will create a PR in the [`ansible-documentation`
+   repository](https://github.com/ansible/ansible-documentation/).The release manager needs to check the Porting Guide PR once manually and un-draft to change the status to 'ready to review.' Afterwards, the [ansible-community/release-management-wg
```

Comment:
Not to be too pedantic here but "un-draft" isn't a word. You could probably drop that and say:

"and change the status to"

Reply:
Also is there a purpose to say "once manually"? Why do we need to specify the number of times?